ddtrace/contrib/starlette/patch.py

```diff
@@ -10,16 +12,58 @@
 log = get_logger(__name__)
 
 config._add(
-    "starlette", dict(_default_service="starlette", request_span_name="starlette.request", distributed_tracing=True)
+    "starlette",
+    dict(
+        _default_service="starlette",
+        request_span_name="starlette.request",
+        distributed_tracing=True,
+        aggregate_resources=True,
+    ),
 )
 
 
+def traced_init(wrapped, instance, args, kwargs):
+    global routes
```

Comment:
Let's try to use starlette's routes directly if possible instead of maintaining our own references 🙂